### DISCUSSION TITLE
Feature Request: Ignore downloaded srt files that are less than 200 bytes

### DISCUSSION THREAD

**Discussion by 007revad** [pymedusa/Medusa#11634] [Ideas & Feature requests]
Lately Medusa has been downloading empty srt files from opensubtitles that only contain 4 lines:
```
1
00:00:00,001 --> 04:00:00,003
Become OpenSubtitles.org VIP Member 
to get subtitles -> osdb.link/vip
```
When I go to the opensusbtitles website and search for the show the subtitles for the effected episode don't exist yet.

It looks like opensubtitles is returning the above srt file when Medusa searches for subtitles that don't exist on opensubtitles. This means Medusa thinks it has downloaded the subtitles already and won't download the real srt file when they later become available.

**Comment by Frank071** [comment]
The opensubtitles.org API has been shutdown since beginning of 2024. This response is the result of that - their way of 'informng' people to move to the new API (opensubtitles.com).
